tests/dialects/test_spark.py

```diff
@@ -129,6 +129,36 @@ def test_ddl(self):
                 "spark": "ALTER TABLE StudentInfo DROP COLUMNS (LastName, DOB)",
             },
         )
+        self.validate_all(
+            "ALTER VIEW StudentInfoView AS SELECT * FROM StudentInfo",
```

Comment:
Let's make all of these `validate_identity`.

Reply:
fixed